packages/bot-skeleton/src/services/api/ticks_service.js

```diff
@@ -54,8 +57,12 @@ export default class TicksService {
             return Promise.resolve(this.pipSizes);
         }
 
+        if (!this.active_symbol_promise) {
+            this.active_symbol_promise = this.api.getActiveSymbolsBrief();
+        }
+
```

Comment:
This is good, but we should return the existing `Promise` if `this.active_symbols_promise` is not undefined, currently you're returning a new `Promise` regardless of whether we have one in memory. It will work, but why create an extra `Promise` if we don't have to?

Reply:
done